@astancovTT on WH device perf stable diffusion tests also runs `pytest models/experimental/stable_diffusion_xl_base/tests/test_sdxl_perf.py -m models_device_performance_bare_metal`

It fails on BH, should it be fixed or skipped?
https://github.com/tenstorrent/tt-metal/actions/runs/17806465230/job/50620826017#step:8:2648
```
FAILED models/experimental/stable_diffusion_xl_base/tests/test_sdxl_perf.py::test_sdxl_unet_perf_device - subprocess.CalledProcessError: Command '["python3 -m tracy -p -r -o /work/generated/profiler/sdxl_unet --check-exit-code  -t 5000 -m 'pytest models/experimental/stable_diffusion_xl_base/tests/test_sdxl_perf.py::test_unet'"]' returned non-zero exit status 4.
```

--- a/.github/workflows/perf-device-models-impl.yaml
+++ b/.github/workflows/perf-device-models-impl.yaml
@@ -99,7 +99,7 @@ jobs:
           device_perf_model_name: "stable_diffusion"
           commands: |
             pytest models/demos/${{ matrix.test-info.arch == 'blackhole' && 'blackhole' || 'wormhole' }}/stable_diffusion/tests -m models_device_performance_bare_metal --timeout=600
-            pytest models/experimental/stable_diffusion_xl_base/tests/test_sdxl_perf.py -m models_device_performance_bare_metal
+            ${{ matrix.test-info.arch == 'wormhole_b0' && 'pytest models/experimental/stable_diffusion_xl_base/tests/test_sdxl_perf.py -m models_device_performance_bare_metal' || '' }}
         if: ${{ !cancelled() && matrix.test-info.civ2-compatible && (matrix.test-info.job-set1 || matrix.test-info.job-blackhole-set1) }}
         timeout-minutes: ${{ matrix.test-info.timeout }}
 

